Docker based development for Jekyll (#7273)
opensearch-project/documentation-website: CONTRIBUTING.md

If you're adding a new page or making major changes to the documentation, such as adding new images, sections, or styling, we recommend that you work in a local copy of the repository and test the rendered HTML before submitting a PR. 

> #### Setting up your Docker based copy of the repository

Reviewer: ```suggestion
#### Setting up a Docker-based copy of the repository

```

> This assumes you have Docker installed.

Reviewer: ```suggestion
If you have Docker installed, you can use Docker to test any major changes to the documentation website locally by using the following steps:
```

> 1. [Fork this repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and clone your fork.

Reviewer: ```suggestion
1. [Fork this repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and clone your fork in the directory of your choice.
```

1. Navigate to your cloned repository.

> 1. Build the documentation website Jekyll image via: `docker build -t documentation-website .`

Reviewer: ```suggestion
1. Build the documentation website Jekyll image by using the following command: 

    ```
    docker build -t documentation-website .
    ```
```

> 1. Start Jekyll via: `docker run -p 4000:4000 -v $(pwd):/app documentation-website`

Reviewer: ```suggestion
1. Start Jekyll from inside the image using the following command: 

   ```
   docker run -p 4000:4000 -v $(pwd):/app documentation-website
   ```
```

> Be aware that it takes a while to build the site the first time, eventually you will see `Server running... press ctrl-c to stop`.

Reviewer: ```suggestion
When building the site for the first time, it'll take some time for the site to build. When your Jekyll instance is ready, you will see `Server running... press ctrl-c to stop`.
```
